### PR TITLE
relock xsoar-tools for validatecontent

### PR DESCRIPTION
**Pull request by JudahSchwartz**
Will lock on the new demisto-sdk version after it is released

### CODE REVIEW

**Comment by JudahSchwartz** [comment]
potentially dependant on https://github.com/demisto/demisto-sdk/pull/2941

**Comment by JudahSchwartz** [comment]
this pr wont be merged pointing to the git commit. will wait for the demisto-sdk pr to be merged to master and released to pypi

File: docker/deprecated_images.json [hunk 1]
```diff
@@ -163,5 +163,10 @@
         "image_name": "demisto/zabbix",
         "reason": "Use the demisto/py3-tools docker image instead.",
         "created_time_utc": "2022-05-31T17:58:02.056066Z"
+    },
+    {
+        "image_name": "demisto/xsoar-tools-hf",
+        "reason": "\"The image is no longer in use by content\"",
```
Comment (JudahSchwartz) [inline docker/deprecated_images.json:169]:
```suggestion
        "reason": "The image is no longer in use by content",
```

**Comment by ShahafBenYakir** [comment]
Ping me once the version is released and updated here

File: docker/xsoar-tools/Pipfile [hunk 2]
```diff
@@ -4,8 +4,8 @@ verify_ssl = true
 name = "pypi"
 
 [packages]
-demisto-sdk = "*"
 demisto-py = "*"
+demisto-sdk = "==1.14.*" # this package should be updated manually. No reason this is locked
```
Comment (ShahafBenYakir) [inline docker/xsoar-tools/Pipfile:8]: As we discussed
```suggestion
demisto-sdk = "==*" # this package should be updated manually. No reason this is locked
```

**Review by ShahafBenYakir — APPROVED** [review]

**Comment by JudahSchwartz** [comment]
forcing since the failing build is deprecated

**Pull request merged**